scripts/globals/fishing_contest.lua

```diff
@@ -0,0 +1,417 @@
+-----------------------------------
+-- Data for Fishing
```

Comment:
If this is _just_ the data, and there's a driver somewhere else the whole file can/should be `fishing_constest_data.lua`